Git-Discard-Unstaged-Changes.md

```diff
@@ -0,0 +1,13 @@
+### Discard unstaged changes
+
+To discard unstaged file changes, one can use the 'git checkout' command to do so. 
+
+To unstage one file: 
+    
+    git checkout <path to file>
```

Comment:
Also mention that user should not use `<path-to-file>` verbatim. Rather, ask them to replace it with the actual file name.